public/js/actions/sources.js

```diff
@@ -51,6 +52,19 @@ function newSource(source) {
   };
 }
 
+function loadSourceMapSources(tab, source, dispatch) {
+  loadSourceMap(tab, source).then(map => {
+    const originalUrls = getOriginalSources(map);
+    let inc = 1;
```

Comment:
The generated id is a sequential id, not a uuid. A uuid is guaranteed to never conflict, so if in a test we load a snapshot of the state and add a new sourcemapped source, that will still work, whereas here it will potentially conflict because `inc` will start from scratch.